### ISSUE TITLE
name is a reserved map name

### ISSUE THREAD

**Issue by g3rd** [emehrkay/Pypher#11]
When I do:

```
p.With(__.collect(__.Distinct(__.map_projection('u', '.*',  name=__.apoc_text_join(__.List(__.u.__first_name__, __.u.__last_name__), "' '")))))
```

I get:

```
__init__() got multiple values for argument 'name'
```

**Comment by emehrkay** [comment]
I think it is because you’re doing `name=`. In MapProjection, the name argument is what appears outside of the map `name {.field,...}` so in your case it is ‘u’. Try to drop it

**Comment by emehrkay** [comment]
Ohhh you need name in the resulting map. I wonder if something like this would work 

```python
d = {‘name’: __.apoc(...)}
__.map_projection(‘u’, ‘*’, **d
```

**Comment by emehrkay** [comment]
It doesn’t. I will change the variable name to something like _name. Or maybe get rid of kwargs in Map and Statement.

**Comment by emehrkay** [comment]
This is fixed with 0.9.2

**Comment by g3rd** [comment]
Thanks!
